docs/config/examples/grafanaLoki.md

```diff
@@ -29,6 +30,6 @@ password=<loki_password>
 ## Result
 
 <div style="text-align: center; margin: 20px;">
-    <img src="../../images/grafanaLoki1.png" alt="Grafana Loki 1" style="box-shadow: 0 4px 8px rgba(0, 0, 0, 0.2); border-radius: 8px; max-width: 100%; display: block; margin-bottom: 20px;">
-    <img src="../../images/grafanaLoki2.png" alt="Grafana Loki 2" style="box-shadow: 0 4px 8px rgba(0, 0, 0, 0.2); border-radius: 8px; max-width: 100%;">
+    <img src="https://github.com/ThePrez/Manzan/blob/main/docs/images/grafanaLoki1.png?raw=true" alt="Grafana Loki 1" style="box-shadow: 0 4px 8px rgba(0, 0, 0, 0.2); border-radius: 8px; max-width: 100%; display: block; margin-bottom: 20px;">
```

Comment:
We should specify relative path here as well. Just need to make sure in /docs/assets/images

Reply:
Same as above